manta-crypto/src/merkle_tree/node.rs

```diff
@@ -159,7 +160,7 @@ impl Default for Parity {
     derive(Deserialize, Serialize),
     serde(crate = "manta_util::serde", deny_unknown_fields)
 )]
-#[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq)]
+#[derive(Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Why is the canonical ordering the right one? Maybe explain in the doc for this type.

Reply:
I used `Ord` for an earlier design, it's not needed for the final one, so I just removed it.